src/wp-includes/class-wp-autoloader.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class WP_Autoloader
+ *
+ * WordPress Class, Interface and Trait Autoloader
+ *
+ */
+class WP_Autoloader {
+
+	/**
+	 * @var array
+	 */
+	private static $_classmap;
+
+	/**
+	 * WP_Autoloader constructor.
+	 */
+	function __construct() {
+
+		if ( ! defined( 'DISABLE_CORE_AUTOLOADER' ) ) {
```

Comment:
This seems to be the current version (as I had looked at the collective files changed). You check for existence of the constant, but not what value it was set to.